deploy/nginx/default.conf

```diff
@@ -32,7 +32,7 @@ server {
 
     location /webdav {
         rewrite /webdav/(.*) /$1 break;
-        proxy_pass ${WEBDAV_URL};
+        proxy_pass $WEBDAV_URL;
```

Comment:
why removing the brackets? This used to work and is required if https://serverfault.com/a/755541/511220 is correct